docs/backend/upgrading/migrate-to-volto.md

```diff
@@ -0,0 +1,97 @@
+Test all migrations thoroughly before applying them on a production environment!
+
+A site that is made compatible with Volto will be accessible with Plone Classic UI, but it will behave differently.
+For example, editors can only effectively work with the content using Volto because html is no longer editable in the TinyMCE editor.
+```
+
+The required steps are:
+
+1.  **Install the packages {py:mod}`plone.volto` and {py:mod}`plone.restapi`.**
+
+    {py:mod}`plone.restapi` is the RESTful API for Plone that allows the frontend Volto to communicate with the backend.
+    {py:mod}`plone.volto` configures Plone to work with Volto, the new default frontend for Plone 6.
+
+1.  **Migrate RichText fields to Volto blocks**
+
+    Volto has a new editor called Slate, whereas Classic UI uses TinyMCE.
+    This step converts the HTML stored in RichText fields to text blocks, allowing you to edit them in Volto.
+    Images, links, and most kinds of HTML formatting are preserved.
+
+    For this you need to have `blocks-conversion-tool` running on an accessible URL.
+    The easiest way to have that running on your machine is to use Docker:
+
+    ```shell
+    docker run -it -p 5000:5000 plone/blocks-conversion-tool:latest
+    ```
+
+    For more options read https://github.com/plone/blocks-conversion-tool.
+
+1.  **Pages, News Items, and Events are made folderish**
+
+    That means these types can contain other content such as Images or other Pages.
+    When you create a new site in Plone 6, this setting is also applied automatically.
+    But existing content remains non-folderish until this step is run.
+
+1.  **Turn folders into folderish pages**
+
+    In Volto adding Folders is disabled by default.
+    Instead folderish pages are used to create folder structures.
+    This step turns all folders into folderish pages.
+    If the folder shows a listing of the content, an appropriate listing block will be added.
+    If the folder shows a default page, then it will be handled in the next step.
+    You can re-enable Folders by checking the box *Implicitly addable?* in ``/portal_types/Folder/manage_propertiesForm``.
+
+1.  **Default Pages of Folders are merged with the Folderish Pages that replace the Folder**
+
+    Volto does not have a concept of default pages.
+    Instead folderish pages can show text, a listing of all the content inside that page, or both.
+    This step takes the content of the default page (such as text or the query of a collection), and adds that to the folderish page that replaces the folder.
+    Metadata (subjects, author, rights, and so on) and relations are moved to the folderish page.
+
+
+1.  **Collections are migrated to Pages with Listing Blocks**
+
+    In Volto adding Collections is disabled by default.
+    Instead folderish pages with listing blocks are used.
+    This step turns all collections into folderish pages.
+    The criteria of the collection are uses to configure a listing block in that page.
```

Comment:
uses -> used

Reply:
Fixed in c1d559abe18885f39411b9b9dbb6577512f31111, along with other things to reduce some review cycles.